libraries/joomla/document/jsonapi.php

```diff
@@ -26,7 +26,7 @@ class JDocumentJsonapi extends JDocumentJson implements JsonSerializable
 	 * @var    Document
 	 * @since  __DEPLOY_VERSION__
 	 */
-	protected $document;
+	public $document;
```

Comment:
We need to decide whether to create wrapper methods for everything (like the `addLink` method I'm using in com_content view, or make this public)